sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/TableCatalog.java

```diff
@@ -210,6 +210,10 @@ default Table createTable(
 
   /**
    * Create a table in the catalog.
+   * <p>
+   * @deprecated This is deprecated. Please override
+   * @link #createTable(TableInfo) instead.
```

Comment:
I believe the Javadoc is broken here and the reference is broken here. I also think the deprecate clause is usually is the last one at the bottom.

```
@throws ...
@throws ...
@deprecated Override {@link #createTable(Identifier, TableInfo)} instead.
```

Reply:
I removed the deprecation, but the Javadoc was stale. fixed.